client/src/Components/Nav/Nav.css

```diff
@@ -0,0 +1,54 @@
+.ant-layout-header{
+background: white;
+}
+.ant-menu.ant-menu-dark{
+background: white;
+}
+.ant-menu.ant-menu-dark .ant-menu-item-selected {
+  background-color: #469D62;
+}
+.ant-menu-dark.ant-menu-horizontal > .ant-menu-item:hover {
+  background-color: #469D62;
+  color: black;
+}
+.ant-menu-dark.ant-menu-horizontal > .ant-menu-item:active {
+  background-color: #469D62;
+  color: black;
+}
```

Comment:
When changing directly the style from the antd class then it will be applied to all other components sharing the same class name. so you need to use them but keep the changes to your components